### DISCUSSION TITLE
Shared Form View CORS [Sep 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3805] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can u share your setup  
#  
> 🙋‍♂️ **wkw**  
#### is it running locally? docker? heroku one click?  
#  
> 🙋‍♂️ **emdax**  
#### running on Ubuntu server 20.22  
#  
> 🙋‍♂️ **emdax**  
#### starting from binary , using SystemD  Unit File  
#  
> 🙋‍♂️ **wkw**  
#### weird. the share form view link is using different port?  
#  
> 🙋‍♂️ **emdax**  
#### same port, but I would like to submit to the form end-point directly from my own form on another website  
#  
> 🙋‍♂️ **emdax**  
#### or submit other JSON data directly into the table from another app  
#  
> 🙋‍♂️ **wkw**  
#### i dont get wt u mean by `submit to the form end-point directly `  
#  
> 🙋‍♂️ **wkw**  
#### u embed the form in your website?  
#  
> 🙋‍♂️ **emdax**  
#### no i would like to submit from another front end  
#  
> 🙋‍♂️ **emdax**  
#### my own custom form POST to same url as embedded form  
#  
> 🙋‍♂️ **wkw**  
#### so if i understand correctly, u hv your own frontend form, which posts the data  to `<host>:<port>/api/v1/db/public/shared-view/<viewId>/rows` upon submission right?  
#  
> 🙋‍♂️ **emdax**  
#### yes  
#  
> 🙋‍♂️ **emdax**  
#### 👍🏼  
#  
> 🙋‍♂️ **emdax**  
#### exactly  
#  
> 🙋‍♂️ **emdax**  
#### Any thoughts?  
#  
> 🙋‍♂️ **emdax**  
#### <@495466997304262658> curious your input  
#  
> 🙋‍♂️ **emdax**  
#### your summary is exactly what im looking for  
#  
> 🙋‍♂️ **wkw**  
#### i dont think it s doable atm  
#  
> 🙋‍♂️ **emdax**  
#### Isn't this the most basic use case?  
#  
> 🙋‍♂️ **emdax**  
#### https://github.com/nocodb/nocodb/issues/3521  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016989769743470633)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
